src/mokka/src/main/java/pl/hycom/mokka/emulator/mock/MockConfigurationManager.java

```diff
@@ -115,15 +126,44 @@ private void updatePathcache() {
     }
 
     @Transactional
-    public MockConfiguration saveOrUpdateMockConfiguration(MockConfiguration mock) {
+    public MockConfiguration createMockConfiguration(MockConfiguration mock){
+        if (mock == null) {
+            return mock;
+        }
+
+        mock.setUpdated(new Date(System.currentTimeMillis()));
+
+        MockConfiguration m = repository.save(mock);
+
+        StubMappingAddEvent stubMappingAddEvent = new StubMappingAddEvent(this, mock.getId());
+        applicationEventPublisher.publishEvent(stubMappingAddEvent);
+
+        log.info(MOCK_ID + mock.getId() + ") added");
```

Comment:
logger supports templating. So the following works better: log.info("something {} ", mock.getId())